datacenter/ucp/2.1/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -13,7 +13,18 @@ The next step is creating a backup policy and disaster recovery plan.
 
 ## Backup policy
 
-As part of your backup policy you should regularly create backups of UCP.
+As part of your backup policy you should regularly create backups of swarm and
+UCP.
+
+When you create a UCP backup, you're only saving:
+
+* Users, teams, and permissions.
+* All UCP configuration options available under `Admin Settings`, such as the
+DDC subscription license, scheduling options, Content Trust and authentication
+backends.
+
+So you need to also create a backup of swarm, to make sure you also save
```

Comment:
It's not great to start a sentence with "So". Also you are not backing up "swarm", you are backing up your swarm manager and worker configurations.